src/features/duplicates/components/ConfigureModal.tsx

```diff
@@ -0,0 +1,119 @@
+import theme from 'theme';
+import {
+  Alert,
+  AlertTitle,
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  useMediaQuery,
+} from '@mui/material';
+import { FC, useState } from 'react';
+import React, { useEffect } from 'react';
+
+import FieldSettings from './FieldSettings';
+import messageIds from '../l10n/messageIds';
+import { PotentialDuplicate } from '../store';
+import PotentialDuplicatesLists from './PotentialDuplicatesLists';
+import useDuplicatesMutations from '../hooks/useDuplicatesMutations';
+import useFieldSettings from '../hooks/useFieldSettings';
+import { useMessages } from 'core/i18n';
+import { useNumericRouteParams } from 'core/hooks';
+import { ZetkinPerson } from 'utils/types/zetkin';
+
+interface ConfigureModalProps {
+  potentialDuplicate: PotentialDuplicate;
+  onClose: () => void;
+  open: boolean;
+}
+
+const ConfigureModal: FC<ConfigureModalProps> = ({
+  potentialDuplicate,
+  open,
+  onClose,
+}) => {
+  const { orgId } = useNumericRouteParams();
+  const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
+  const messages = useMessages(messageIds);
+  const { mergeDuplicate } = useDuplicatesMutations(orgId);
+
+  const [selectedIds, setSelectedIds] = useState<number[]>(
+    potentialDuplicate?.duplicates.map((person) => person.id) ?? []
+  );
+
+  const peopleToMerge = potentialDuplicate?.duplicates.filter((person) =>
+    selectedIds.includes(person.id)
+  );
+
+  const peopleNoToMerge = potentialDuplicate?.duplicates.filter(
+    (person) => !selectedIds.includes(person.id)
+  );
+
+  const { fieldValues, initialOverrides } = useFieldSettings(peopleToMerge);
+  const [overrides, setOverrides] = useState(initialOverrides);
+
+  useEffect(() => {
+    setSelectedIds(
+      potentialDuplicate?.duplicates.map((person) => person.id) ?? []
+    );
+  }, [open]);
+
+  return (
+    <Dialog fullScreen={fullScreen} maxWidth={'lg'} open={open}>
+      <DialogTitle variant="h5">{messages.modal.title()}</DialogTitle>
+      <Box display="flex" flexGrow={1} overflow="hidden">
+        <Box
+          display="flex"
+          flexDirection="column"
```

Comment:
I don't see a need for flex here, and in fact I believe it's causing a bug where, if the modal is very small vertically, the lists appear on top of each other.

![image](https://github.com/zetkin/app.zetkin.org/assets/550212/f332a1f1-232a-4710-bc0b-5dfbd6d303df)